webapi/Controllers/DocumentController.cs

```diff
@@ -120,6 +122,52 @@ public class DocumentController : ControllerBase
             documentImportForm);
     }
 
+    /// <summary>
+    /// Service API for removing a document.
+    /// Documents imported through this route will be considered as global documents.
+    /// </summary>
+    [Route("documents")]
+    [HttpDelete]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
```

Comment:
Shouldn't this be an admin-only operation?

Reply:
Thanks, always open to input...but this PR isn't at that level of detail...but yeah, if you look I have did introduce admin concept.